components/automate-chef-io/content/docs/iam-actions.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "IAM Actions"
+description = "Reference Actions for Roles"
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "reference"
+    weight = 60
++++
+
+Reference the chart on this page when creating a *Role* to know which action grants access to what page in the browser.
+
+*IAM Action* lists the associated action or actions required to access that page in the browser.
+
+|  Task           |  IAM Action      | API endpoint  | Browser Page |
```

Comment:
im not sure we need the API endpoint here, since we now have the actions listed in the API docs as well

also, what is the difference between task and browser page?
wonder if we should have the URL instead of either of these?

Reply:
Task should be a brief and human-friendly description, a "how do I do [x]", with x being the description. I think these should be longer, I'm thinking fragments or clauses, but not whole sentences. I think we might want to flip the API and URL columns. The purpose of this index is to help users build out the relationship between Task-Action-Endpoint-URL; a user should be able to take any point of information and discover its correlation across the column.

Reply:
Browser page should be "URL", and we should give them like 
| View events in the event feed      | event:*         | <endpoint>     | {{https://{{< example_fqdn "automate" >}}/dashboards/event-feed |

(this is a guess on of what the elements are & probably not correct)

Reply:
for task its prolly gonna get complicated quickly, many of our pages have a number of tasks that users can do on them. we could leverage the descriptions on those pages though, that appears right below the headings. happy to demo if needed.

Reply:
Great! Set up a demo with @mjingle . We expect repetition for URL & that repetition is even helpful for users to anticipate and remember the associations.